scripts/travis.sh

```diff
@@ -0,0 +1,207 @@
+#!/usr/bin/env bash
```

Comment:
why do we need travis.sh instead of just using apollo.sh? Looks like they are largely identical.